data-reconciliation-app/test/unit-test/repositories/kv-repository.test.ts

```diff
@@ -1,64 +1,74 @@
-// TODO: need to use ccf polyfill to solve the issue of key-value map with unit-test
+// Use the CCF polyfill to mock-up all key-value map functionality for unit-test
 import "@microsoft/ccf-app/polyfill.js";
 
 import { randomUUID } from "crypto";
-import { IRepository } from "../../../src/repositories/kv-repository";
+import keyValueRepository, {
```

Comment:
I think your prettier setting is still 80 characters.

Reply:
I've add it as a separate story